src/com/company/exceptions/NumberOfArgsException.java

```diff
@@ -0,0 +1,19 @@
+package com.company.exceptions;
+
+public class NumberOfArgsException extends Exception{
```

Comment:
Лучше наследоваться от `RuntimeException` - использовать непроверяемые исключения.
Изначально идея проверяемых исключений казалась полезной, но потом от нее ушли и сейчас в основном используют `RuntimeException`.

```suggestion
public class NumberOfArgsException extends RuntimeException{
```